ONSdigital/ssdc-rm-docker-dev
# 126 Unify the RH/RM docker compose files

# Motivation and Context

We're integrating RH into our RM stack, so we can reduce docker dev to a single set of docker compose files again and remove the RM/RH specific commands.

# What has changed



- Move RH UI and dependency into RM docker compose files
- Remove common dependencies file
- Remove redundant Make targets
- Update README

# How to test?



Run `make pull`, `make up` and `make logs`, check the RH UI is now running properly and all the services are still starting up healthy, run `make down` and check everything is stopped. 

# Links



https://trello.com/c/tXXyDXC6/126-add-rh-ui-deployment-to-our-pipelines-5
